src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -108,19 +113,25 @@ public CaseDetailsEndpoint(@Qualifier(CreatorGetCaseOperation.QUALIFIER) final G
     @ApiOperation(value = "Get case", notes = "Retrieve an existing case with its state and data")
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = "Case found for the given ID"),
-        @ApiResponse(code = 400, message = "Invalid case ID"),
+        @ApiResponse(code = 400, message = "Invalid request"),
```

Comment:
We can keep this as "Bad Request" as a standard across CCD, particularly as we already have catalogue response object that contains business reading of this response?